equistore/src/labels.rs

```diff
@@ -135,8 +135,8 @@ pub struct Labels {
 // Labels can be sent to other thread safely since eqs_labels_t uses an
 // `Arc<equistore_core::Labels>`, so freeing them from another thread is fine
 unsafe impl Send for Labels {}
-// &Labels can be sent to other thread safely since there is no interior
-// mutability
+// &Labels can be sent to other thread safely since there is no un-synchronized
+// interior mutability (`user_data` is protected by RwLock).
```

Comment:
As mentioned in the commit message, these functions are not yet exposed in the Rust/Python bindings, since I don't see a use case for them. We can always add them later if we need them!